README.md

```diff
@@ -134,6 +134,27 @@ To open the prometheus web UI, once the container is up, run the below:
 $ make prometheus-ui
 ```
 
+### Start / Stop mock for rbac
+
+- To use this you need to enable RBAC into `config/configs.yaml` file:
+
+  ```yaml
+  clients:
+    rbac_enabled: True
+    rbac_base_url: http://localhost:8800/api/rbac/v1
+    rbac_timeout: 30
+  mocks:
+    my_org_id: "12345"
+    rbac:
+      account_admin: "12345"
+      account_viewer: "123456"
+  ```
```

Comment:
only having a single org have access might be a tad annoying when development.  What about adding some more option:

```
mocks:
  rbac:
    denied_org_ids: ["1","2","3"]
    viewer_org_ids: ["4", "5"]
```

and then the logic could be:

1) if in denied org_ids list, send empty list of permission
2) if in viewer_org_ids, only allow for view only
3) else allow all.

Reply:
and you could set a default in the config.yaml.exmple of 

    denied_org_ids: ["denied"]
    viewer_org_ids: ["view"]